### DISCUSSION TITLE
[BUG] 无论怎么修改config.toml，页面不变

### DISCUSSION THREAD

**Discussion by Yangly0** [HEIGE-PCloud/DoIt#504] [Q&A] [answered]
### Describe the bug 描述你遇到的错误

当我修改config.toml后，页面并不发生变化。

### Expected behavior 期待的行为

解决该问题

### Screenshots 屏幕截图

页面一直出现的样子：
![image](https://user-images.githubusercontent.com/43691492/156787831-a52250b7-37c4-45f3-afbb-2183e2329e9a.png)

config.toml的配置：
![image](https://user-images.githubusercontent.com/43691492/156788136-692d6819-9229-4381-a859-e0100118bf1a.png)

### Build Environment 构建环境

- hugo v0.92.2-CDF6A0D6+extended windows/amd64

### Preview Environment 预览环境

- chrome

### Additional Information 补充信息

PS：我已经删除并重新安装2次，操作如下。然后，我发现打开依然会如上图一样。
```
$ rm -rf my-blog
$ hugo new site my-blog
$ git init  
$ git submodule add https://github.com/HEIGE-PCloud/DoIt.git themes/DoIt
$ hugo server
```

**Comment by WingsZeng** [comment]
看起来您并没有使用 DoIt 主题. 您的配置文件中并没有指定主题. 如果您在构建的时候加了 `-t` 选项指定 DoIt 主题的话, 这样的配置应该是会报错的.

请参考 [该文档](https://hugodoit.pages.dev/zh-cn/theme-documentation-basics/#2-%E5%AE%89%E8%A3%85) 进行配置.

**Reply by Yangly0** [reply]
> 看起来您并没有使用 DoIt 主题. 您的配置文件中并没有指定主题. 如果您在构建的时候加了 `-t` 选项指定 DoIt 主题的话, 这样的配置应该是会报错的.
> 
> 请参考 [该文档](https://hugodoit.pages.dev/zh-cn/theme-documentation-basics/#2-%E5%AE%89%E8%A3%85) 进行配置.

感谢老哥了，其实我已经配置多次DoIt了，昨天突然出现的情况，今天我参照文档进行了基础配置，它显示还是第一次加载的情况（即电脑开机运行`hugo server`，后续再次运行这个命令始终加载以前相同的页面）。应该是hugo的bug，页面刷新不了，我打算去官网问问。

**Comment by Yangly0** [comment, marked as answer]
我已经解决了这个bug，因为我的chrome有代理[SwitchyOmega](chrome-extension://padekgcemlokbadohgkifijomclgjgif/options.html#!/about)，所以每次无法加载最新显示，选择直连就可以解决。
